i18n/en.json

```diff
@@ -15,7 +15,10 @@
 	"wikibasereconcileedit-editendpoint-qualifiers-references-not-supported": "Qualifiers and References are not currently supported",
 	"wikibasereconcileedit-editendpoint-reconciliation-property-missing-in-statements": "Entity must have at least one statement for the reconciliation Property",
 	"wikibasereconcileedit-editendpoint-invalid-reconciliation-statement-type": "Reconciliation statement must contain a value",
+	"wikibasereconcileedit-editendpoint-invalid-request-body": "Invalid or empty JSON request body",
+	"wikibasereconcileedit-editendpoint-invalid-content-type": "Content-type should be application/json",
```

Comment:
Hm, maybe the expected content type should be a message parameter?

Reply:
sure!